### PR TITLE
Feature ETP-768: Change <option> HTML tag improving a patch.

### PR DESCRIPTION
**Pull request by Matias-Bernal**
ETP-768: Applied a patch to improve <option> HTML tag in <selection> tag to improve the code and prepare it to future upgrade to remove <span> tags into <options> and avoid a Chrome warning.

### CODE REVIEW

File: src/org/openbravo/erpCommon/ad_reports/ReportAnnualCertification.html [hunk 1]
```diff
@@ -259,7 +259,8 @@
               <td class="TitleCell"><span class="LabelText">Organization</span></td>
               <td class="Combo_ContentCell"> 
                 <select  name="inpOrg" id="inpOrg" class="Combo Combo_TwoCells_width" required="false">
-                  <option value=""> <div id="reportAD_ORGID"></div></option>
+                  <option value=""/>
+				  <div id="reportAD_ORGID"/>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportAnnualCertification.html:263]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html [hunk 2]
```diff
@@ -241,13 +241,16 @@
...
 	              <td class="TitleCell"><span class="LabelText">Organization</span></td>
-	              <td class="Combo_ContentCell"> <select  name="inpOrg" id="inpOrg" class="ComboKey Combo_TwoCells_width" required="true">
-	                 <option value=""><div id="reportAD_ORG_ID"></div></option>
-	                </select></td>
-				  <td class="ContentCell"></td>
-				  <td class="ContentCell"></td>
-				  <td class="ContentCell"></td>
-				  <td class="ContentCell"></td>             
+	              <td class="Combo_ContentCell">
+                  <select  name="inpOrg" id="inpOrg" class="ComboKey Combo_TwoCells_width" required="true">
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html:245]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html [hunk 3]
```diff
@@ -310,14 +313,20 @@
              
             <tr>
 	              <td class="TitleCell"><span class="LabelText">Asset Category</span></td>
-	              <td class="Combo_ContentCell"> <select  name="inpcAssetCategoryId" id="inpcAssetCategoryId" class="Combo Combo_TwoCells_width">
-                  <option value=""> <div id="reportA_ASSET_GROUP_ID"></div></option>
-                </select></td>	              
-	                <td class="ContentCell"></td>
+	              <td class="Combo_ContentCell">
+                  <select  name="inpcAssetCategoryId" id="inpcAssetCategoryId" class="Combo Combo_TwoCells_width">
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html:317]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html [hunk 4]
```diff
@@ -310,14 +313,20 @@
...
+                    <div id="reportA_ASSET_GROUP_ID"/>
+                  </select>
+                </td>
+	              <td class="ContentCell"></td>
 	              <td class="TitleCell"><span class="LabelText">Account Schema</span></td>
-	              <td class="Combo_ContentCell"> <select  name="inpcAcctSchemaId" id="inpcAcctSchemaId" class="Combo Combo_TwoCells_width">
-                  <option value=""> <div id="reportC_ACCTSCHEMA_ID"></div></option>
-                </select></td>
+	              <td class="Combo_ContentCell">
+                  <select  name="inpcAcctSchemaId" id="inpcAcctSchemaId" class="Combo Combo_TwoCells_width">
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportAssetDepreciationSchedule.html:325]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportSalesOrderInvoicedJasper.html [hunk 5]
```diff
@@ -355,20 +356,29 @@
...
-                      <option value=""> <div id="reportM_PRODUCT_CATEGORYID"></div></option>
-                    </select></td>
-		<td class="TitleCell"><span class="LabelText">Product</span></td>
+                  <td class="Combo_ContentCell" colspan="2">
+                    <select name="inpProductCategory" id="inpProductCategory2" class="Combo Combo_TwoCells_width">
+                      <option value=""/>
+                      <div id="reportM_PRODUCT_CATEGORYID"/>
+                    </select>
+                  </td>
+		              <td class="TitleCell"><span class="LabelText">Product</span></td>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportSalesOrderInvoicedJasper.html:381]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportPurchaseDimensionalAnalysesFilterJR.html [hunk 6]
```diff
@@ -836,7 +845,8 @@
                   <td class="TitleCell"><span class="LabelText">Currency</span></td>
                   <td class="Combo_ContentCell">
                   	<select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
-                    <option value=""><div id="reportC_Currency_ID"></div></option>
+                      <option value=""/>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportPurchaseDimensionalAnalysesFilterJR.html:848]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportProjectBuildingSiteJR.html [hunk 7]
```diff
@@ -326,23 +326,30 @@
...
                   <td class="Combo_ContentCell">
-  		     		<select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
-     		      	<option value=""><div id="reportC_Currency_ID"></div></option>
-   		     		</select>
-		   		  </td>
+                  <select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
+                    <option value=""/>
+                    <div id="reportC_Currency_ID"/>
+                  </select>
+                </td>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportProjectBuildingSiteJR.html:336]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportOrderNotInvoiceFilterJR.html [hunk 8]
```diff
@@ -320,15 +321,22 @@
...
-                      <option value=""> <div id="reportAD_ORGID"></div></option>
-                    </select></td>
-		<td class="TitleCell"><span class="LabelText">Details</span></td>
+                  <td class="Combo_ContentCell" colspan="2">
+                    <select name="inpOrg" id="inpOrg" class="Combo Combo_TwoCells_width">
+                      <option value=""/>
+                      <div id="reportAD_ORGID"/>
+                    </select>
+                  </td>
+		              <td class="TitleCell"><span class="LabelText">Details</span></td>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportOrderNotInvoiceFilterJR.html:339]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportInvoicesJR.html [hunk 9]
```diff
@@ -301,15 +301,19 @@
                   <td class="TitleCell"><span class="LabelText">Currency</span></td>
                   <td class="Combo_ContentCell">
   		     		<select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
-     		      	<option value=""><div id="reportC_Currency_ID"></div></option>
+                <option value=""/>
+                <div id="reportC_Currency_ID"/>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportInvoicesJR.html:305]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportInvoiceVendorFilterJR.html [hunk 10]
```diff
@@ -301,9 +301,10 @@
                   <td class="TitleCell"><span class="LabelText">Currency</span></td>
                   <td class="Combo_ContentCell">
                   	<select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportInvoiceVendorFilterJR.html:303]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportInvoiceVendorDimensionalAnalysesFilterJR.html [hunk 11]
```diff
@@ -853,7 +865,8 @@
                     <td class="TitleCell"><span class="LabelText">Currency</span></td>
                   	<td class="Combo_ContentCell">
                   	  <select name="inpCurrencyId" id="inpCurrencyId" class="ComboKey Combo_OneCell_width" required="true">
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportInvoiceVendorDimensionalAnalysesFilterJR.html:867]: @Matias-Bernal fix indentation

File: src/org/openbravo/erpCommon/ad_reports/ReportExpense.html [hunk 12]
```diff
@@ -356,20 +356,21 @@
...
                 </tr>
-
-				<tr>
-					 <td class="TitleCell"> <span class="LabelText">Employee</span></td>
-	                   <td class="Combo_ContentCell" colspan="2"> 
-	                    <select name="inpUser" id="inpUser" class="Combo Combo_TwoCells_width" >
-	                      <option value=""></option>
-	                        <div id="reportC_BPartner_ID"></div>
-	                    </select></td>
+				        <tr>
```
Comment (valeg-etendo) [inline src/org/openbravo/erpCommon/ad_reports/ReportExpense.html:359]: @Matias-Bernal fix indentation

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

**Review by RomanMagnoli — APPROVED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Pull request merged**